modules/http/src/main/java/org/mule/module/http/internal/request/DefaultHttpRequesterConfig.java

```diff
@@ -111,7 +113,15 @@ public void initialise() throws InitialisationException
                 .setOwnerName(name)
                 .build();
 
-        httpClient = new GrizzlyHttpClient(configuration);
+        GrizzlyHttpClientFactory httpClientFactory = muleContext.getRegistry().get(HTTP_CLIENT_FACTORY_NAME);
```

Comment:
This is okay here, since it is is the initialize() method and not the constructor.